From 6dac286145838e914631ee556e38aeb5f821d835 Mon Sep 17 00:00:00 2001
From: jamborm <jamborm@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Thu, 15 May 2014 12:16:33 +0000
Subject: [PATCH] 2014-05-15  Martin Jambor  <mjambor@suse.cz>

	PR ipa/60897
	* ipa-prop.c (ipa_modify_formal_parameters): Reset DECL_LANG_SPECIFIC.



git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_9-branch@210472 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/ChangeLog  |    5 +++++
 gcc/ipa-prop.c |    1 +
 2 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index 33d9eec..5fec565 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,8 @@
+2014-05-15  Martin Jambor  <mjambor@suse.cz>
+
+	PR ipa/60897
+	* ipa-prop.c (ipa_modify_formal_parameters): Reset DECL_LANG_SPECIFIC.
+
 2014-05-15  Jakub Jelinek  <jakub@redhat.com>
 
 	PR tree-optimization/61158
diff --git a/gcc/ipa-prop.c b/gcc/ipa-prop.c
index 9f144fa..0bc44d3 100644
--- a/gcc/ipa-prop.c
+++ b/gcc/ipa-prop.c
@@ -3650,6 +3650,7 @@ ipa_modify_formal_parameters (tree fndecl, ipa_parm_adjustment_vec adjustments)
 
   TREE_TYPE (fndecl) = new_type;
   DECL_VIRTUAL_P (fndecl) = 0;
+  DECL_LANG_SPECIFIC (fndecl) = NULL;
   otypes.release ();
   oparms.release ();
 }
-- 
1.7.1

